tests/performance/MemoryTestBase.h

```diff
@@ -22,13 +22,13 @@
 #include <gtest/gtest.h>
 
 #include <olp/core/cache/CacheSettings.h>
+#include <olp/core/cache/KeyValueCache.h>
 #include <olp/core/client/OlpClientSettings.h>
 #include <olp/core/client/OlpClientSettingsFactory.h>
 #include <olp/core/utils/Dir.h>
 #include <testutils/CustomParameters.hpp>
 
 #include "NetworkWrapper.h"
```

Comment:
If you already touch this, then please do it properly. All headers should be grouped together as none are system headers.